core/domain/summary_services.py

```
@@ -704,73 +683,84 @@ def get_library_groups(language_codes: List[str]) -> List[LibraryGroupDict]:
-        for summary in exp_fetchers.get_exploration_summaries_matching_ids(
-            all_exp_ids
+        # Query explorations from the datastore by category and language.
+        exp_models_result: Sequence[exp_models.ExpSummaryModel] = (
+            exp_models.ExpSummaryModel.query()
+            .filter(
+                exp_models.ExpSummaryModel.status
+                == constants.ACTIVITY_STATUS_PUBLIC
+            )
+            .filter(
+                exp_models.ExpSummaryModel.deleted  # pylint: disable=singleton-comparison
+                == False
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `False`; use `not exp_models.ExpSummaryModel.deleted:` for false checks

Replace with `not exp_models.ExpSummaryModel.deleted`

(E712)